packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -97,6 +97,7 @@ class UserMessages {
   String androidLicensesUnknown(Platform platform) =>
       'Android license status unknown.\n'
       'Try re-installing or updating your Android SDK Manager.\n'
+      'Maybe you can use `flutter doctor --android-licenses` to solve this problem.\n'
       'See https://developer.android.com/studio/#downloads or visit '
       '${_androidSdkInstallUrl(platform)} for detailed instructions.';
```

Comment:
Good idea!  How about totally removing the suggestion to re-install the SDK manager.
```suggestion
      'Android license status unknown.\n'
      'Run `flutter doctor --android-licenses` to accept the SDK licenses.\n'
      'See ${_androidSdkInstallUrl(platform)} for more details.';
```

Reply:
Thank you very much for supporting my idea! this suggestion has been committed!